testsuite/tests/kuadrant/authorino/identity/extended_properties/test_extended_properties.py

```diff
@@ -0,0 +1,49 @@
+"""Basic tests for extended properties"""
+import pytest
+
+from testsuite.objects import Value, ValueFrom
+from testsuite.utils import extract_response
+
+
+@pytest.fixture(scope="module")
+def authorization(authorization, rhsso):
+    """
+    Add new identity with list of extended properties. This list contains:
+        - Static `value` and dynamic `jsonPath` properties
+        - Dynamic chaining properties which point to another extended property location before its created
+    Add simple response to inspect 'auth.identity' part of authJson where the properties will be created.
+    """
+    authorization.identity.oidc(
+        "rhsso",
+        rhsso.well_known["issuer"],
+        extended_properties=[
+            Value("static", name="property_static"),
+            # ValueFrom points to the request uri
+            ValueFrom("context.request.http.path", name="property_dynamic"),
+            ValueFrom("auth.identity.property_static", name="property_chain_static"),
+            ValueFrom("auth.identity.property_dynamic", name="property_chain_dynamic"),
+            ValueFrom("auth.identity.property_chain_self", name="property_chain_self", overwrite=True),
+        ],
+    )
+    authorization.responses.add_simple("auth.identity")
+    return authorization
+
+
+def test_basic(client, auth):
+    """
+    This test checks if static and dynamic extended properties are created and have the right value.
+    """
+    response = client.get("/anything/abc", auth=auth)
+    assert extract_response(response)["property_static"] % "" == "static"
```

Comment:
"NOT_SET", "MISSING" or any label o this kind would be bit more helpful during test results review than empty string

Reply:
Good idea!